### PR TITLE
Fix ORTModelForSeq2SeqLM test

### PR DESCRIPTION
**Pull request by fxmarty**
# What does this PR do?

Fix a test that was not passing.

### CODE REVIEW

File: tests/onnxruntime/test_modeling.py [hunk 1]
```diff
@@ -1386,8 +1386,10 @@ def test_pipeline_on_gpu(self, model_arch):
         # check model device
         self.assertEqual(pipe.model.device.type.lower(), "cuda")
         # compare model output class
-        self.assertTrue(isinstance(outputs[0]["generated_text"], str))
-        self.assertTrue(len(outputs[0]["generated_text"]) > len(text))
+        self.assertTrue(isinstance(outputs[0]["translation_text"], str))
+
+        # does not pass with hf-internal-testing/tiny-random-t5 , but OK with t5-base
+        # self.assertTrue(len(outputs[0]["translation_text"]) > len(text))
```
Comment (philschmid) [inline tests/onnxruntime/test_modeling.py:1392]: why is not passing?

File: tests/onnxruntime/test_modeling.py [hunk 2]
```diff
@@ -1386,8 +1386,10 @@ def test_pipeline_on_gpu(self, model_arch):
         # check model device
```
Comment (philschmid) [inline tests/onnxruntime/test_modeling.py:1386]: should be added 1 line above: 
We can also add `generation_args` to have a `min_lenght` for the test. to pass?

File: tests/onnxruntime/test_modeling.py (hunk 1 above)
Comment (fxmarty) [reply tests/onnxruntime/test_modeling.py:1392]: no idea, it's not passing either with `hf-internal-testing/tiny-random-t5` + transformer model

Comment (fxmarty) [reply tests/onnxruntime/test_modeling.py:1392]: The test was not run up to know due to the require gpu decorator

Comment (philschmid) [reply tests/onnxruntime/test_modeling.py:1392]: can you check why? like whats the issue?

Comment (fxmarty) [reply tests/onnxruntime/test_modeling.py:1392]: The output `translation_text` is `""`. I just think it's the model being bad.

Comment (philschmid) [reply tests/onnxruntime/test_modeling.py:1392]: even when setting explicitly `min_length`?

**Comment by fxmarty** [comment]
Yup, could be a bug in transformers.

```python
from transformers import pipeline as tf_pipeline
from transformers import AutoModelForSeq2SeqLM, AutoTokenizer

model_id = "hf-internal-testing/tiny-random-t5"

model = AutoModelForSeq2SeqLM.from_pretrained(model_id)
tokenizer = AutoTokenizer.from_pretrained(model_id)

pipe = tf_pipeline("translation_en_to_de", model=model, tokenizer=tokenizer, min_length=len(text) + 1, max_length=2 * len(text) + 1)
outputs = pipe(text)

print(outputs)
print(len(outputs[0]["translation_text"]) > len(text))

## prints
[{'translation_text': ''}]
False
```

I'll look more into it ASAP.

File: tests/onnxruntime/test_modeling.py (hunk 1 above)
Comment (fxmarty) [reply tests/onnxruntime/test_modeling.py]: When the pipeline parameter  `return_tensors` is `False`, the padding tokens are removed from the output. Hence the issue. Fixed now.

**Review by michaelbenayoun — APPROVED** [review]
Great, thanks for fixing this!

**Pull request merged**